### PR TITLE
entry point compatible with aiida v2 data type name

### PR DESCRIPTION
**Pull request by unkcpz**

### CODE REVIEW

File: .make_ghpages/generate_json.py [hunk 1]
```diff
@@ -51,9 +51,29 @@ def parse_yaml_files_in_folder(folder_path, exclude=None):
...
+
+    return res
+
+final_dict_v2 = copy.deepcopy(final_dict)
+
+# Loop over or the fields and update to compatible with aiida 2.x entry points name
+for domain in final_dict_v2:
+    for computer in final_dict_v2[domain]:
+        if computer != 'default':
+            final_dict_v2[domain][computer]["computer-setup"] = update_to_v2_entry_points(final_dict_v2[domain][computer]["computer-setup"])
```
Comment (yakutovicha) [inline .make_ghpages/generate_json.py:70]: I would rather run something like this here:
```suggestion
            update_to_v2_entry_points(final_dict_v2[domain][computer]["computer-setup"])
```
and remove `res = copy.deepcopy(comp_setup)` line in `update_to_v2_entry_points`. The dictionary is already| "deep copied", so this it is safe to modify it directly here.

File: .make_ghpages/generate_json.py [hunk 2]
```diff
@@ -51,9 +51,29 @@ def parse_yaml_files_in_folder(folder_path, exclude=None):
...
+
 # Store the extracted information as a single JSON file.
 os.mkdir(folder_path/'out')
 with open(folder_path/'out/database.json', 'w') as filep:
     json.dump(final_dict, filep, indent=4)
 
-
+# Stroe the v2 compatible entry points
+with open(folder_path/'out/database_v2.json', 'w') as filep:
+    json.dump(final_dict_v2, filep, indent=4)
```
Comment (yakutovicha) [inline .make_ghpages/generate_json.py:79]:
```suggestion
    json.dump(final_dict_v2, filep, indent=4)

```

File: .make_ghpages/generate_json.py [hunk 3]
```diff
@@ -51,9 +51,29 @@ def parse_yaml_files_in_folder(folder_path, exclude=None):
...
 
+# Prepare the config db for aiida 2.x data type entry points compatibility
+def update_to_v2_entry_points(comp_setup: dict) -> dict:
+    # v1 -> v2 with attach `core.` in front for transport and scheduler
+    res = copy.deepcopy(comp_setup)
+    for k, v in comp_setup.items():
+        if k in ['transport', 'scheduler']:
+            res[k] = '.'.join(['core', v])
+
+    return res
```
Comment (yakutovicha) [inline .make_ghpages/generate_json.py:62]:
```suggestion
    for key in ['transport', 'scheduler']:
        try:
            comp_setup[key] = f"core.{comp_setup[key]}"
        except KeyError:
            print(f"No {key} key specified for {comp_setup['label']}")
```

**Review by yakutovicha — CHANGES_REQUESTED** [review]
Thanks, @unkcpz, I have a couple of suggestions.

**Comment by unkcpz** [comment]
@yakutovicha thanks! changed.

**Review by yakutovicha — APPROVED** [review]
All good, thanks!

**Comment by yakutovicha** [comment]
> One way is to set these parameters for all config yaml files and let 1.x ignore them.

sure can you make a PR for this?

**Pull request merged**